src/core/ext/filters/client_channel/lb_policy/weighted_round_robin/weighted_round_robin.cc

```diff
@@ -511,6 +512,9 @@ WeightedRoundRobin::Picker::Picker(
       subchannels_.emplace_back(sd->subchannel()->Ref(), sd->weight());
     }
   }
+  global_stats().IncrementWrrSubchannelListSize(
+      subchannel_list->num_subchannels());
+  global_stats().IncrementWrrSubchannelListSize(subchannels_.size());
```

Comment:
This should be `SubchannelReadySize`, not `SubchannelListSize`.

Reply:
done